ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsDetailsScreen.tsx

```diff
@@ -70,6 +81,30 @@ const PaymentsTransactionBizEventsDetailsScreen = () => {
     );
   };
 
+  const handleOnDownloadPdfReceiptError = () => {
+    IOToast.error("Impossibile scaricare la ricevuta, riprova più tardi.");
```

Comment:
Since we are in a function component we should use the `useIOToast` hook instead.

Reply:
You're right! I also realized that there were missing some translations and with this commit (4cc9a97df5b46923025021f07d2e64ad00cb1d18) was solved